src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java

```diff
@@ -580,35 +580,27 @@ public void testDestroyCheckerWithWrongCacheFileNameLength() throws Exception {
 
     /**
      * It is OK to have long test method name here as it describes the test purpose.
+     * @throws java.lang.Exception
      */
     @Test
     public void testCacheAndCheckWhichDoesNotImplementExternalResourceHolderInterface()
             throws Exception {
         assertWithMessage("ExternalResourceHolder has changed his parent")
                 .that(ExternalResourceHolder.class.isAssignableFrom(HiddenFieldCheck.class))
                 .isFalse();
-        final DefaultConfiguration checkConfig = createModuleConfig(HiddenFieldCheck.class);
-
-        final DefaultConfiguration treeWalkerConfig = createModuleConfig(TreeWalker.class);
-        treeWalkerConfig.addChild(checkConfig);
-
-        final DefaultConfiguration checkerConfig = createRootConfig(treeWalkerConfig);
-        checkerConfig.addProperty("charset", StandardCharsets.UTF_8.name());
-
         final File cacheFile = File.createTempFile("junit", null, temporaryFolder);
-        checkerConfig.addProperty("cacheFile", cacheFile.getPath());
 
         final File tmpFile = File.createTempFile("file", ".java", temporaryFolder);
         final String[] expected = CommonUtil.EMPTY_STRING_ARRAY;
 
-        verify(checkerConfig, tmpFile.getPath(), expected);
+        verifyWithInlineConfigParser(tmpFile.getPath(), expected);
```

Comment:
You can not simply change method, you need to add to input file check that was added for execution in Junit. Please review other changes that were done for this issue to get more examples

Reply:
perfect, thanks so much for the suggestions, I'll get to it straight away